File: JuliaLang/ExaWizards/ExaWizards_2019_A_v01.jl
# [contests]   ExaWizards 2019
# [task]       A  
# [URL]        https://atcoder.jp/contests/exawizards2019/tasks/exawizards2019_a
# [compiler]   Julia (0.5.0)
# [submission] https://atcoder.jp/contests/exawizards2019/submissions/4788797 : AC
# [reference]  https://qiita.com/phigasui/items/2db20b36fb85e161e0ae

function exawizards2019_A()

    # STEP.01
    # read out the given integers
    A, B, C = Base.parse.(Base.split(Base.readline(Base.STDIN)))

    # STEP.02
    # determine if there exists an equilateral triangle whose sides have lengths A, B and C.
    if Base.isequal(A, B) && Base.isequal(B, C)
        Base.println(Base.STDOUT, "Yes")
    else
        Base.println(Base.STDOUT, "No")
    end

    # STEP.TRUE_END
    return Base.nothing

end

#=======================#
# Main Process is below #
#=======================#

Main.exawizards2019_A()

#==================#
# End of this code #
#==================#

File: C(GCC5.4.1)/ExaWizards/ExaWizards_2019_A_v01.c
// [contests]   ExaWizards 2019
// [task]       A
// [URL]        https://atcoder.jp/contests/exawizards2019/tasks/exawizards2019_a
// [compiler]   C (GCC 5.4.1)
// [submission] https://atcoder.jp/contests/exawizards2019/submissions/4788618 : AC

#include <stdio.h>


int main (void)
{
    /* variables for main process */
    unsigned char A, B, C;

    /* support variables for main process */
    int retval_scanf;

    // STEP.01
    // read out the given integers
    retval_scanf = scanf("%hhu %hhu %hhu", &A, &B, &C);

    // STEP.02
    // determine if there exists an equilateral triangle whose sides have lengths A, B and C.
    if (A == B && B == C) { printf("%s\n", "Yes"); }
    else                  { printf("%s\n", "No" ); }
    
    // STEP.TRUE_END
    return 0;
}
